Extension-Example.md

```diff
@@ -7,7 +7,7 @@ Hypothetical API
 // one promise-returning object.
 
 const neededPermissions = {
-  accounts: { method: 'eth_accounts', caveats: [ { limit: 1 } ] },
+  accounts: { method: 'eth_requestAccounts', caveats: [ { limit: 1 } ] },
```

Comment:
This is kindof a different thing. `eth_requestAccounts` is arguably a totally different approach to requesting permission to login from rpc-cap, but the method isn't equivalent to `eth_accounts`, because `eth_accounts` is not expected to initiate a user approval prompt, while `requestAccounts` is designed to initiate an approval prompt.

So under the rpc-cap model, `eth_accounts` (a simple read) is restricted, and will return `AUTH_ERROR` _until_ approval is presented in our more standard form.

The `requestAccounts` model would suggest that we eventually implement a custom method for requesting each type of permission we decide to manage.